.github/workflows/build-artifacts-code.yml

```diff
@@ -62,5 +62,5 @@ jobs:
           path: |
             src/out/*
       - name: Publish Nuget packages to GitHub registry
-        working-directory: src
-        run: dotnet nuget push out/* -k ${{secrets.GITHUB_TOKEN}}
+        working-directory: src/out
```

Comment:
So, now you can even keep `src` as current dir if you like. Then the current dir will be the same throughout the script.

Reply:
Yeah; it shouldn't matter but I'll change it back; hold on

Reply:
I just hadn't tested it this way.

Reply:
Done